common/djangoapps/student/signals/receivers.py

```diff
@@ -34,6 +39,33 @@ def on_user_updated(sender, instance, **kwargs):
         # Check for a retired email.
         if is_email_retired(instance.email):
             raise AccountValidationError(
-                EMAIL_EXISTS_MSG_FMT.format(username=instance.email),
+                EMAIL_EXISTS_MSG_FMT.format(email=instance.email),
```

Comment:
This was just... a broken line before? `EMAIL_EXISTS_MSG_FMT` was just never defined anywhere. So I fixed it while I was near this code.

Reply:
Well that answers my above question. But also whaaaat.